server/src/logger/opts.ts

```diff
@@ -0,0 +1,8 @@
+const SimpleNodeLogger = require('simple-node-logger'),
+  opts = {
+    logFilePath: 'mylogfile.log',
+    timestampFormat: 'YYYY-MM-DD HH:mm:ss.SSS',
+  },
+  log = SimpleNodeLogger.createSimpleLogger(opts);
```

Comment:
Fajnie by było to zrobić by gdy nie jesteśmy na produkcji logował normalnie do konsoli jak dotychczas